test/Microsoft.ML.TestFramework/Microsoft.ML.TestFramework.csproj

```diff
@@ -9,7 +9,7 @@
     <ProjectReference Include="..\..\src\Microsoft.ML.Core\Microsoft.ML.Core.csproj" />
     <ProjectReference Include="..\..\src\Microsoft.ML.Data\Microsoft.ML.Data.csproj" />
     <ProjectReference Include="..\..\src\Microsoft.ML.FastTree\Microsoft.ML.FastTree.csproj" />
-    <ProjectReference Include="..\..\src\Microsoft.ML.Maml\Microsoft.ML.Maml.csproj" />
+    <ProjectReference Include="..\..\src\Microsoft.ML.Maml\Microsoft.ML.Console.csproj" />
```

Comment:
>Console [](start = 72, length = 7)

Just so I'm clear, could you explain why this solution works? My understanding from the explanation earlier is we needed to produce a separate project, because some projects were referencing `Maml.csproj`, and something couldn't be both referenced as a project and be invokable. (I have no idea whether that's actually correct.) But now I see we seemingly can. So why split `Maml.csproj` at all?

Reply:
Ah OK. Because it's a .NET Core, and this test assembly is *also* .NET core. :P This scheme is subideal. I'd like to keep `Maml.cs` with those other files.

Don't do this please, just leave `Maml.csproj` the way it is. (By which I mean, the files.) Instead just have a single additional framework that's .NET core that calls into that DLL.

---
In reply to: [191826072](https://github.com/dotnet/machinelearning/pull/248#discussion_r191826072) [](ancestors = 191826072)